app/config/api.py

```diff
@@ -0,0 +1,11 @@
+from ninja import NinjaAPI
+
+api = NinjaAPI()
+
+
+@api.get('/checker')
+def checker(request):
+    return {"success": True, "message": " OK"}
+
+
+api.add_router("/provider", "provider.api.router")
```

Comment:
I guess the path should be `/providers` or `/api/providers`. Not sure where to best add the `/api` prefix, as it will be used by all apps.

Reply:
I think it would also be better to import the router and add it as an object as it will make any refactoring/renaming easier in the future.